modules/light-clients/08-wasm/types/vm.go

```diff
@@ -10,20 +10,13 @@ import (
 
 	errorsmod "cosmossdk.io/errors"
 
-	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-)
 
-var (
-	WasmVM *wasmvm.VM
-	// Store key for 08-wasm module, required as a global so that the KV store can be retrieved
-	// in the ClientState Initialize function which doesn't have access to the keeper.
-	// The storeKey is used to check the code hash of the contract and determine if the light client
-	// is allowed to be instantiated.
-	WasmStoreKey  storetypes.StoreKey
-	VMGasRegister = NewDefaultWasmGasRegister()
```

Comment:
I could also move `VMGasRegister` to the internal package, but that requires reorganizing more code. Let me know if you think it's worth.

Reply:
in a followup pr makes sense to me, then we could swap it out for the wasmd maintained one if a separate go.mod is ever made there (without breaking api)